### PR TITLE
.

### PR DESCRIPTION
**Pull request by alper-han**
.

### CODE REVIEW

File: pkgs/os-specific/linux/r8125/default.nix [hunk 1]
```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
       A kernel module for Realtek 8125 2.5G network cards.
     '';
     # r8125 has been integrated into the kernel as of v5.9.1
-    broken = lib.versionAtLeast kernel.version "5.9.1";
+    #broken = lib.versionAtLeast kernel.version "5.9.1";
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/r8125/default.nix:42]: What's up with this? If it is false, it should be removed

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) when naming your commits.

File: pkgs/os-specific/linux/r8125/default.nix (hunk 1 above)
Comment (alper-han) [reply pkgs/os-specific/linux/r8125/default.nix]: Thanks for the information

**Comment by alper-han** [comment]
@SuperSandro2000 

No need to update, I will use manual from now on.
It is good that you encourage others to learn to use manual.

**Pull request closed without merging**